### PR TITLE
Makes bloody footprints messier

### PR DESCRIPTION
**Pull request by Lufferly**
## About The Pull Request

Makes bloody shoes loose less blood per step, meaning that they trail more blood. Updates the equation for determining footprint alpha to account for this change.
(Bloodiness of 50, 100, 150 from left to right)
Before:
image
After:
image

## Why It's Good For The Game

Right now footprint go like 2-3 steps before stopping, feels kinda lame.
🕵️ 

## Changelog

:cl: Seven
balance: Bloody footprint trails go a longer distance now.
/:cl:

### CODE REVIEW

File: code/datums/components/bloodysoles.dm [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/// The percentage of blood we lose on each step
+#define BLOOD_PERCENT_LOSS_ON_STEP 0.25
```
Comment (MrMelbert) [inline code/datums/components/bloodysoles.dm:2]: When I was tweaking this for another project, I went with `0.33`, and found it to be a pretty happy range. I suggest moving it up to that.

**Review by MrMelbert — COMMENTED** [review]
Awesome

**Comment by SomeoneYouProbablyKnow** [comment]
this feels way too consistent. i agree that footprints should go further but a big line leading where you went is kinda silly (it's not even that strong but it is silly). Maybe add random-ish breaks in the line, would be so much more gratifying to find the continuation of the trail past an airlock or something

File: code/datums/components/bloodysoles.dm (hunk 1 above)
Comment (Lufferly) [reply code/datums/components/bloodysoles.dm]: done

**Pull request merged**